### ISSUE TITLE
模型转换之后检测结果与原模型检测结果不一致

### ISSUE THREAD

**Issue by junqiCao** [Tencent/ncnn#2230]
如题，我按照官方的讲解 https://zhuanlan.zhihu.com/p/128974102 ，尝试了yolact的转换，转换过程中都比较顺利，能够在ncnn中调用模型，但是检测结果不对，类型和准确率基本一致，但是检测框显示的是全图，未准确获取实例分割的区域。
系统环境：Ubuntu16.04
pytorch1.4.0
ncnn版本为最新的20200413
onnx版本 1.6.0
onnx simplier 0.2.6
onnx runtime 1.2.0

简化的模型图如下：
Permute          759                      1 1 758 759 0=3
Reshape          769                      1 1 759 769 0=2 1=-1
Convolution      770                      1 1 745_splitncnn_0 770 0=96 1=3 11=3 2=1 12=1 3=1 13=1 4=1 14=1 15=1 16=1 5=1 6=221184
Permute          771                      1 1 770 771 0=3
Reshape          781                      1 1 771 781 0=32 1=-1
UnaryOp          782                      1 1 781 782 0=16
_Concat           783                      5 1 601 640 679 718 757 783 0=0
Concat           784                      5 1 613 652 691 730 769 784 0=0
Concat           785                      5 1 626 665 704 743 782 785 0=0
Softmax          787                      1 1 784 787 0=1 1=1_

_Convolution      585                      1 1 584 585 0=32 1=1 11=1 2=1 12=1 3=1 13=1 4=0 14=0 15=0 16=0 5=1 6=8192_
ReLU             586                      1 1 585 586
Permute          587                      1 1 586 587 0=3

斜体字体为实际使用的模型的参数

程序如下：
ex.extract("585", maskmaps); // 138x138 x 32

ex.extract("783", location);   // 4 x 19248
ex.extract("785", mask);       // maskdim 32 x 19248
ex.extract("787", confidence); // 81 x 19248

不知道是什么问题导致这样的结果，求官方帮忙看一下是哪里的问题，谢谢

**Comment by Byronnar** [comment]
> 5
你好，请问这个问题当初是怎么解决的呢，我也遇到了一样的问题.. 能解答一下么

**Comment by nihui** [comment]
针对onnx模型转换的各种问题，推荐使用最新的pnnx工具转换到ncnn
In view of various problems in onnx model conversion, it is recommended to use the latest pnnx tool to convert your model to ncnn

```shell
pip install pnnx
pnnx model.onnx inputshape=[1,3,224,224]
```

详细参考文档
Detailed reference documentation
https://github.com/pnnx/pnnx
https://github.com/Tencent/ncnn/wiki/use-ncnn-with-pytorch-or-onnx#how-to-use-pnnx
